<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window_add</class>
 <widget class="QMainWindow" name="Window_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adding</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_back">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&lt;--  Back</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string> Task:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_task"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string> Due Date: (MM/DD/YYYY)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="de_duedate"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string> Person:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_person"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_add">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
